website/content/docs/discovery/dns.mdx

```diff
@@ -285,6 +285,50 @@ caching of DNS results. However, there are many situations in which caching is
 desirable for performance and scalability. This is discussed more in the tutorial
 for [DNS caching](https://learn.hashicorp.com/tutorials/consul/dns-caching).
 
+## DNS with ACLs
+
+After enabling [Access Control Lists (ACLs)](/docs/security/acl/acl-system),
+DNS requests will fail with error `NXDOMAIN` unless ACL tokens have been
+configured with the necessary rules.
+
+When a DNS request is made to a Consul agent, the agent will use a previously
+configured token with the request (since there is no means for the initial DNS
+request to specify a token):
+
+- The agent's [`default` token](/docs/agent/options#acl_tokens_default)
+  token, if configured.
+- Otherwise, the built-in [`anonymous` token](/docs/security/acl/acl-system#builtin-tokens).
```

Comment:
```suggestion
- The built-in [`anonymous` token](/docs/security/acl/acl-system#builtin-tokens), if a default token is not configured.
```

Reply:
Not against this change, but trying to understand it. The previous wording felt more concise to me and clear from the first word, whereas here it's longer and I have to read to the last clause to understand there's a precedence here.

Reply:
This second phrasing feels more explicit and is also a more complete sentence.

Reply:
Changing to something that I think is clearer based on a conversation we had on the network segments tutorial about order of precedence for joining.

```
Consul agents resolve DNS requests using a preconfigured tokens, listed in
order of precedence:

1. The agent's [`default` token](/docs/agent/options#acl_tokens_default).
2. The built-in [`anonymous` token](/docs/security/acl/acl-system#builtin-tokens).
   Because the anonymous token is used when any request is made to Consul without
   explicitly specifying a token, production deployments should not apply policies
   needed for DNS to this token.
```